src/main/java/com/ldtteam/vanillaplustools/ModHammerItem.java

```diff
@@ -1,28 +1,17 @@
 package com.ldtteam.vanillaplustools;
 
 import net.minecraft.world.item.Item;
-import net.minecraft.world.item.ItemStack;
-import net.minecraft.world.item.PickaxeItem;
-import net.minecraft.world.item.Tier;
-import net.neoforged.neoforge.common.ItemAbilities;
-import net.neoforged.neoforge.common.ItemAbility;
-import org.jetbrains.annotations.NotNull;
+import net.minecraft.world.item.ToolMaterial;
 
-public class ModHammerItem extends PickaxeItem
+public class ModHammerItem extends Item

```

Comment:
Is there a reason that PickaxeItem was removed but ShovelItem was not? It's better if there's at least a consistent API usage here, so either:
- Use PickaxeItem and ShovelItem
- Use props.pickaxe and props.shovel
- Keep it as is in case neither of these API's fully exist